Sources/Remote Logging/Crash Logging/CrashLogging.swift

```diff
@@ -56,7 +62,7 @@ public class CrashLogging {
 
         #if DEBUG
         DDLogDebug("📜 This is a debug build")
-        let shouldSendEvent = UserDefaults.standard.bool(forKey: "force-crash-logging")
+        let shouldSendEvent = UserDefaults.standard.bool(forKey: "force-crash-logging") && !dataProvider.userHasOptedOut
```

Comment:
Tests were failing without this change; the tests were trying to verify whether `userHasOptedOut` was working, but in this DEBUG branch, it wasn't ever being looked at.

Reply:
Whoa nice catch!